packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/cart-api/add-address.tsx

```diff
@@ -0,0 +1,32 @@
+
+import React from 'react';
+import {
+  reactExtension,
+  useApi,
+  Tile
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const SmartGridTile = () => {
+  const api = useApi();
```

Comment:
```suggestion
  const api = useApi<'pos.home.tile.render'>();
```

I think if we're doing TS examples we should use the type helper here to get the right api inference.